pkg/fs/fs_test.go

```diff
@@ -0,0 +1,241 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fs
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestTempDir(t *testing.T) {
```

Comment:
this doesn't seem very useful, this unit test is just testing `os.TempDir` ...

if we want to unit test this, we need to retool so we can actually test the behavior we're alterting

Reply:
we should not unit test the go standard library, they have their own unit tests